wire-ios-data-model/Source/MLS/MessageProtocol.swift

```diff
@@ -54,4 +54,15 @@ public enum MessageProtocol: Int16 {
             return nil
         }
     }
+
+    public var stringValue: String {
+        switch self {
+        case .proteus:
+            return "proteus"
+        case .mls:
+            return "mls"
+        case .mixed:
+            return "mixed"
+        }
+    }
```

Comment:
Should we use `enum MessageProtocol: String, RawRepresentable` instead of custom mapping here. You will see errors in `ZMConversation+MLS.swift` to get the primitive value from core data, but we can either:
1. do the mapping in place in `get` and `set`.
2. migrate the core data values to strings to (custom migration required).

Reply:
I briefly tried that, however, I considered it out of scope of my ticket.
@agisilaos you touched this enum recently, what do you think?

Reply:
done: 3b96887e8907b60f6ac17797f7e37a489fc5d7b2